dashboard/app/models/course_offering.rb

```diff
@@ -214,6 +214,7 @@ def summarize_for_assignment_dropdown(user, locale_code)
         display_name: any_versions_launched? ? localized_display_name : localized_display_name + ' *',
         category: category,
         is_featured: is_featured?,
+        assignable: assignable?,
```

Comment:
is this still necessary?

Reply:
Was a remnant of a past approach - removed!